sway/config/output.c

```diff
@@ -478,17 +481,7 @@ bool apply_output_config(struct output_config *oc, struct sway_output *output) {
 		output->max_render_time = oc->max_render_time;
 	}
 
-	// Reconfigure all devices, since input config may have been applied before
```

Comment:
I think it makes sense to keep this comment here

Reply:
I moved (and reworded, since now we care about both enabling/disabling where the comment had been written for enabling only) the comment to https://github.com/swaywm/sway/pull/5468/files#diff-4f65f4327e987fef8ec3796cdb07644eR429-R430; I think that branch contains the more tricky `map_to_output` logic the comment refers to (though this call to `input_manager_configure_all_inputs` is still necessary).

Reply:
I think it makes sense to have both comments. The branch above handles the case of disabling the input since the output is now disabled. This call here handles (re-)enabling the input due to the output becoming available (again).

Reply:
Alright, restored.

Reply:
Thanks.